cmd/lncli/commands.go

```diff
@@ -2142,6 +2142,11 @@ var queryRoutesCommand = cli.Command{
 			Name:  "use_mc",
 			Usage: "use mission control probabilities",
 		},
+		cli.Uint64Flag{
+			Name: "outgoing_chanid",
```

Comment:
nit: add "(optional)" to the description?

Reply:
done